### ISSUE TITLE
select field content omit bug

### ISSUE THREAD

**Issue by wubinhong** [sequelpro/sequelpro#2283]
When I query a select clause in the Query tab, I found that I can only select a part of content for some field with long text. 
For example:
```
SELECT imgsrc FROM `cm_salon_itemcomment` WHERE `order_ticket_id`=85;
```
I got the field imgsrc content like this:
```
[{"img":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/54a546bb65006.jpg","thumbimg":"http:\/\/newapi.choumei.cn\/Uploads\/choumeic...
```
What I expect is:
```
[{"img":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/54a546bb65006.jpg","thumbimg":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/s_54a546bb65006.jpg","width":640,"height":854,"thumbwidth":200,"thumbheight":200},{"img":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/54a546bb65d9b.jpg","thumbimg":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/s_54a546bb65d9b.jpg","width":640,"height":854,"thumbwidth":200,"thumbheight":200},{"img":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/54a546bb66b09.jpg","thumbimg":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/s_54a546bb66b09.jpg","width":640,"height":854,"thumbwidth":200,"thumbheight":200},{"img":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/54a546bb675af.jpg","thumbimg":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/s_54a546bb675af.jpg","width":640,"height":854,"thumbwidth":200,"thumbheight":200},{"img":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/54a546bb680d8.jpg","thumbimg":"http:\/\/newapi.choumei.cn\/Uploads\/choumeicomment\/2015-01-01\/s_54a546bb680d8.jpg","width":640,"height":854,"thumbwidth":200,"thumbheight":200}]
```

By the way, My sequelpro version is 1.1 Build 4499, I never encounter this problem before I upgrade the software to version 1.1.

**Comment by dmoagx** [comment]
To clarify:
* This happens only in the Query editor results table
* The edit has to start in an inline field
* Then use tab to go to a TEXT/BLOB field
* Use cmd+C to copy the value **without** opening the field editor
* Value is truncated after 150 bytes

**Comment by wubinhong** [comment]
Thanks for your reply, for my situation(v1.1), there is no editor pop up
when I click the field in the Query editor results table, so I can copy the
entire content for the TEXT/BLOB field. Would you please fix this up in the
next version?

Thank you very much!

**Comment by mifz** [comment]
Same for me.

select CONCAT("it's very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very very long string");

In results: "it's very very very very very very very very very very very very very very very very very very very very very very very very very very very very very ..."

Version: 1.1.0.1 Build 4500 (2d9145d)

**Comment by dmoagx** [comment]
Issue fixed in 1.1.1
See http://www.sequelpro.com/blog/2016.02/sequel-pro-1-1-1/
